packages/gatsby/src/utils/get-latest-apis.js

```diff
@@ -0,0 +1,29 @@
+const path = require(`path`)
+const fs = require(`fs-extra`)
+const axios = require(`axios`)
+
+const API_FILE = `https://unpkg.com/gatsby/apis.json`
```

Comment:
Nit: I'd put this behind a gatsby.dev url (gatsby.dev/apis.json) for future proofing and if we ever move this to something else

Reply:
Yeah -- not sure. The main drawback is that _sometimes_ `.dev` domains are blocked due to some localhosts hackery, which would mean the file never downloads.

I think keeping it like this is a little safer. I don't think it's likely we'll need to re-name this file, either?

I get what you're doing here, just not sure I'm fully on board!

Reply:
Yeah, I'm likely overthinking this. The file isn't mission critical anyway and is a progressive enhancement so you're probably right in keeping things simple!

Looking great otherwise 🙌

Reply:
It was (and still may be!) a good idea! Just want to try and think of all the trade offs, good and bad!